### PR TITLE
period-2 grading

### PR DESCRIPTION
**Pull request by bwhm**
Please review:
@DzhideX 
- That it works :)
- That it's properly sorted, and I didn't forget anything!

@blrhc 
- That the totaldirect scores per round is correct
- Some spotchecks perhaps?

The last one to approve merges!

### CODE REVIEW

**Review by blrhc — APPROVED** [review]

**Comment by blrhc** [comment]
Numbers seem correct!

File: data/fm-info.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
-{
-    "scoringPeriodsFull":
+"scoringPeriodsFull":
```
Comment (DzhideX) [inline data/fm-info.json:1]: We're missing that opening bracket here for it to be valid. Do you mind updating?

**Review by DzhideX — CHANGES_REQUESTED** [review]
Only one requested change for now. I can give a test again after we update this!

File: data/fm-info.json (hunk 1 above)
Comment (bwhm) [reply data/fm-info.json]: Fixed! 
(?)

**Comment by DzhideX** [comment]
Works great locally. Merging!

**Pull request merged**